aspnetcore/security/index.md

```diff
@@ -32,3 +42,9 @@ ASP.NET Core and EF contain features that help you secure your apps and prevent
 * [Open redirect attacks](xref:security/preventing-open-redirects)
 
 There are more vulnerabilities that you should be aware of. For more information, see the other articles in the **Security and Identity** section of the table of contents.
+
+## Additional resources
+
+* <xref:security/authentication/identity>
+* <xref:security/authentication/identity-enable-qrcodes>
+* <xref:security/authentication/social/index>
```

Comment:
@mairaw these additional links complete everything mentioned under 
# Build secure web apps
at https://dotnet.microsoft.com/en-us/apps/aspnet/web-apps

You can link to this page rather than making them search.